### DISCUSSION TITLE
Letterbox during breaks: layer change?

### DISCUSSION THREAD

**Discussion by CloneWith** [ppy/osu#30076] [Ideas] [answered]
On the latest version of osu!lazer, when `Letterbox during breaks` is on for a beatmap, a gradient shade would appear at the top and botton of the playfield during breaks. However it masks the HUD elements and make them less visible, which isn't like ones in osu!stable.

An example of letterboxes can be seen below.

![Letterbox shade masks HUD elements](https://github.com/user-attachments/assets/84f8bcdd-fcb4-4ae0-a8db-f8ca2d9da052)

I think it can be better to put the letterboxes *under* the HUD layer.

**Comment by felipemarins** [comment, marked as answer]
Already reported: https://github.com/ppy/osu/discussions/29788
